### ISSUE TITLE
1.21.2+ Updates

### ISSUE THREAD

**Issue by A5ho9999** [A5ho9999/MinecraftMods#22]
1.21.2/3 updates are in progress. All mods should be updated over the next week

**Comment by robschof** [comment]
Hello!  love your mod.  Will it be coming to 1.21.4?  thanks :)

**Comment by A5ho9999** [comment]
Yes they will, eventually. As owo-lib has updated today I can begin working on 1.21.4 over the next couple weeks.
